packages/react-ui/src/components/custom/markdown.tsx

```diff
@@ -19,26 +21,48 @@ function applyVariables(markdown: string, variables: Record<string, string>) {
 type MarkdownProps = {
   markdown: string | undefined;
   variables?: Record<string, string>;
-  withBorder?: boolean;
+  markdownType?: MarkdownType;
 };
 
 const Container = ({
-  withBorder,
+  markdownType,
   children,
 }: {
-  withBorder: boolean;
+  markdownType?: MarkdownType;
   children: React.ReactNode;
-}) =>
-  withBorder ? (
-    <Alert className="rounded">
+}) => {
+  let containerClass = '';
+  switch (markdownType) {
+    case 'Info':
+      containerClass = 'rounded';
+      break;
+    case 'Warning':
+      containerClass = 'bg-[#FDF8F2] border-[#FDEBC6] text-[#835300]';
+      break;
+    case 'Tip':
+      containerClass = 'bg-[#F5FEFA] border-[#C7F1E2] text-[#064E3B]';
```

Comment:
did you try to use existing colors?

bg-warning-100 and bg-warning-300?
same for tip which would be success